File: 3-Patron/php/Textura.php
<?php
class Textura{
	private $imagen;
	private $alto;
	private $ancho;

	public function __construct($imagen, $alto, $ancho){
    $this->imagen = $imagen;
    $this->alto = $alto;
    $this->ancho = $ancho;
	}

	public function getImagen(){
		return $this->imagen;	
	}

	public function setImagen($imagen){
		$this->imagen = $imagen;
	}

	public function getAlto(){
		return $this->alto;	
	}

	public function setAlto($alto){
		$this->alto = $alto;
	}

	public function getAncho(){
		return $this->ancho;	
	}

	public function setAncho($ancho){
		$this->ancho = $ancho;
	}

}
?>

File: 3-Patron/Java/Textura.java
package buildercomposite;

public class Textura{
    private Imagen Imagen;
    private int Ancho;
    private int Alto;
    
    public Textura(Imagen imagen, int alto, int ancho){
        this.Imagen = imagen;
        this.Alto = alto;
        this.Ancho = ancho;
    }
    
    public Imagen getImagen(){
        return this.Imagen;
    }
    public void setImagen(Imagen imagen){
        this.Imagen = imagen;
    }
    public int getAlto(){
        return this.Alto;
    }
    public void setAlto(int alto){
        this.Alto = alto;
    }
    public int getAncho(){
        return this.Ancho;
    }
    public void setAncho(int ancho){
        this.Ancho = ancho;
    }
    
}
